### ISSUE TITLE
Wrongly grouped radio buttons in settings (only a11y issue)

### ISSUE THREAD

**Issue by kingalg** [TryQuiet/quiet#1597]
Settings -> Notifications -> both groups of radio buttons
**Priority**: low, it’s implemented incorrectly and can be confusing but it’s still possible to reach active elements.
They can’t be navigated correctly from a11y point of view. You can see how it should work here: https://www.w3.org/WAI/tutorials/forms/grouping/  Most important: group buttons using <fieldset>

**How it should work:** 
Once you turn on the screen reader you should be able to navigate to a group of radio buttons (for example with TAB) and in this group move between elements with up and down arrows. 
Every time the user navigates to a group of buttons they should be notified about the general description of the group. In our cases it will be “Notify me about…”  and “Sound when receiving a notification”. 
When inside of this group the user should be able to move between radio buttons with up and down arrows.
With TAB or SHIFT+TAB the user can navigate between groups of radio buttons or other elements on the page.
**How it’s working in Quiet:**
When the user navigates to the radio button they don’t hear the description of the group so they can’t be sure what is the reason behind the button.
User moves between each radio button with a TAB. In order to check one of the radio buttons they must use space. 

<img width="286" alt="Screenshot 2023-06-27 at 16 24 19" src="https://github.com/TryQuiet/quiet/assets/68909168/1c81dcf1-801f-4c1b-b326-6a441cee406c">
